README.md

```diff
@@ -42,8 +42,7 @@ match '/saml/logout' => 'saml_idp#logout', via: [:get, :post, :delete]
 Create a controller that looks like this, customize to your own situation:
 
 ``` ruby
-class SamlIdpController
-  include SamlIdp::IdpController
+class SamlIdpController < SamlIdp::IdpController
```

Comment:
@pkarman either way the documentation is still wrong though right because the doc say
```ruby
class SamlIdpController
  include SamlIdp::IdpController`
```
on including concern it should be like this
class SamlIdpController
  include SamlIdp::Controller`
```

but know ppl won't be able to inherit the boilerplate code part code.